scriptlets/sru-helpers/fetch_manifest

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+#
+# Copyright 2024 Canonical Ltd.
+#
+# Fetch the manifest for a given machine from C3
+#
+# Usage:
+#     `fetch_manifest <CID> <MACHINE_SECURE_ID>`
+
+from argparse import ArgumentParser
+import configparser
+import requests
+
+
+# This is the manifest that all machines will use if we cannot find
+# one specifically created for that machine
+fallback_manifest_raw = """[manifest]
+com.canonical.certification::has_bt_smart = false
+com.canonical.certification::has_camera = true
+com.canonical.certification::has_card_reader = true
+com.canonical.certification::has_ethernet_adapter = true
+com.canonical.certification::has_thunderbolt = false
+com.canonical.certification::has_thunderbolt3 = false
+com.canonical.certification::has_touchscreen = false
+com.canonical.certification::has_tpm2_chip = false
+com.canonical.certification::has_usb_storage = true
+com.canonical.certification::has_usb_type_c = false
+com.canonical.certification::has_wlan_adapter = true
+"""
+
+fallback_manifest_config = configparser.ConfigParser(delimiters=("=",))
+fallback_manifest_config.read_string(fallback_manifest_raw)
+
+
+def parse_json(manifest_json):
+    """
+    Given an manifest json file, parse it as a format suitable for the Checkbox launcher file
+    """
+    manifest_config = configparser.ConfigParser(delimiters=("=",))
+    manifest_config["manifest"] = manifest_json
```

Comment:
I think it would make sense if the endpoint reply wasn't a shallow dict with all the key-value pairs but rather a dict that contained the `manifest` field you are now manually adding, along with the key-value pairs underneath that. This is would also allow you to add extra fields (other than the `manifest`) to the endpoint's response at a later point, if required.

Reply:
Following the discussion on MM, I think we should keep the simple structure for now, until we have a large collection of manifests before we actually start adding more features on top of the manifest data.